python/ray/dataframe/test/test_dataframe.py

```diff
@@ -222,6 +225,10 @@ def test_float_dataframe():
     test_idxmax(ray_df, pandas_df)
     test_idxmin(ray_df, pandas_df)
     test_pop(ray_df, pandas_df)
+    test_iterrows(ray_df, pandas_df)
+    test_items(ray_df, pandas_df)
+    test_iteritems(ray_df, pandas_df)
+    test_itertuples(ray_df, pandas_df)
```

Comment:
I believe there is another test in the current master. Please rebase and add these to that test.